requirements.txt

```diff
@@ -5,7 +5,7 @@ wsaccel==0.6.7  # recommended for acceleration of gevent-websocket. But abandone
 web3==7.4.0
 eth-typing==5.0.1
 content-hash==2.0.0
-rotki-pysqlcipher3==2024.1.2
+rotki-pysqlcipher3==2024.10.1
```

Comment:
Can you revert my assertion commit also? Instead of assertions now we should test with this and that this works. Right?